Source/OxyPlot/Annotations/TextualAnnotation.cs

```diff
@@ -31,7 +33,7 @@ protected TextualAnnotation()
         /// Gets or sets the annotation text.
         /// </summary>
         /// <value>The text.</value>
-        public string Text { get; set; }
+        public string? Text { get; set; }
```

Comment:
File will need further changes when `MathRenderingExtensions` is updated: might be worth doing that now?